public/default/index.html

```diff
@@ -4,28 +4,64 @@
     />
     <meta name="robots" content="index, follow" />
     <meta name="author" content="Vocdoni" />
-    <meta property="og:title" content="The voice of digital voting" />
+
+    <!-- Open Graph Protocol for Facebook -->
+    <meta property="og:title" content="Vocdoni App: Simplifying Secure Digital Voting for Everyone" />
     <meta
       property="og:description"
-      content="Voting platform for allowing to create Web2/Web3 voting processes and let your community decide"
+      content="Vocdoni App makes digital voting accessible and straightforward for organizations and governments, streamlining the voting process with a focus on security, user experience, and cost-effectiveness, enabling votes from anywhere on any device."
     />
     <meta property="og:image" content="/logo512.png" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <link rel="apple-touch-icon" href="/logo128.png" />
-    <link rel="manifest" href="/manifest.json" />
-    <link rel="canonical" href="https://app.vocdoni.io" />
-    <title>Vocdoni - The voice of digital voting</title>
+    <meta property="og:url" content="https://app.vocdoni.io">
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="Vocdoni App: Simplifying Secure Digital Voting for Everyone">
+
+    <!-- Twitter Card -->
+    <meta  property="twitter:title" content="Vocdoni App: Simplifying Secure Digital Voting for Everyone" />
+    <meta
+      content="Discover how Vocdoni App revolutionizes digital voting with a user-friendly, secure, and cost-effective platform, perfect for organizations and governments aiming to simplify decision-making and engage communities."
+      property="twitter:description"
+    />
+    <meta property="twitter:image" content="/assets/vocdoniapp.png" />
+    <meta property="og:type" content="website" />
+    <meta content="summary_large_image" name="twitter:card" />
+    <meta property="og:url" content="https://app.vocdoni.io">
+    <meta property="og:type" content="website">
+    <meta name="twitter:site" content="@vocdoni">
+    <meta name="twitter:creator" content="@vocdoni">
+
+    <!-- LinkedIn -->
+    <meta property="og:title" content="Vocdoni App: Simplifying Secure Digital Voting for Everyone">
+    <meta property="og:description" content="Vocdoni App is leading the way in digital voting, offering a secure, GDPR-compliant, and user-friendly platform that simplifies the voting process for organizations and local governments. Find out how at https://vocdoni.app.">
+    <meta property="og:image" content="/assets/vocdoniapp.png">
+    <meta property="og:url" content="https://app.vocdoni.io">
+    <meta property="og:type" content="website">
```

Comment:
I see lots of repeated properties. i.e. `og:type` is defined three times. Same for `og:url`... properties are unique

Reply:
Resolved.